play25-common/src/main/scala/com/typesafe/conductr/lib/play/api/ConnectionHandler.scala

```diff
@@ -0,0 +1,45 @@
+package com.typesafe.conductr.lib.play.api
+
+import com.google.inject.Inject
+import com.typesafe.conductr.lib.HttpPayload
+import com.typesafe.conductr.lib.scala.{ AbstractConnectionContext, AbstractConnectionHandler }
+import _root_.play.api.libs.ws._
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+/**
+ * When performing Play.WS connections, this is the connection context to use.
+ */
+class ConnectionContext @Inject() (val wsClient: WSClient, implicit val executionContext: ExecutionContext)
```

Comment:
This means that we don't set `compressionEnabled` to true and instead use the default `WSClient`?